src/gui/guiFormSpecMenu.cpp

```diff
@@ -2855,26 +2855,15 @@ void GUIFormSpecMenu::drawList(const ListDrawSpec &s, int layer,
 			}
 
 			// Draw tooltip
-			std::wstring tooltip_text;
 			if (hovering && !m_selected_item) {
-				const std::string &desc = item.metadata.getString("description");
-				if (desc.empty())
-					tooltip_text =
-						utf8_to_wide(item.getDefinition(m_client->idef()).description);
-				else
-					tooltip_text = utf8_to_wide(desc);
-
-				if (!item.name.empty()) {
-					if (tooltip_text.empty())
-						tooltip_text = utf8_to_wide(item.name);
-					else if (m_tooltip_append_itemname)
-						tooltip_text += utf8_to_wide("\n[" + item.name + "]");
+				std::string tooltip = item.getDescription(m_client->idef());
+				if (!tooltip.empty()) {
```

Comment:
this will always be true since getDescription never returns an empty description

Reply:
getDescription returns an empty description when the stack is empty. I guess there's no need to get the tooltip in that case.